src/traces/contour/attributes.js

```diff
@@ -106,7 +107,33 @@ module.exports = extendFlat({}, {
             role: 'style',
             description: [
                 'Determines whether or not the contour lines are drawn.',
-                'Has only an effect if `contours.coloring` is set to *fill*.'
+                'Has an effect only if `contours.coloring` is set to *fill*.'
+            ].join(' ')
+        },
+        showlabels: {
+            valType: 'boolean',
+            dflt: false,
+            role: 'style',
+            description: [
+                'Determines whether to label the contour lines with their values.'
+            ].join(' ')
+        },
+        font: extendFlat({}, fontAttrs, {
+            description: [
+                'Sets the font used for labeling the contour levels.',
+                'The default color comes from the lines, if shown.',
+                // TODO: same size as layout.font, or smaller? 80%?
```

Comment:
I played with that a little, and ended up feeling like that was a little small:
![screen shot 2017-06-26 at 4 57 18 pm](https://user-images.githubusercontent.com/2678795/27560012-a4b9f3d4-5a90-11e7-8d1d-63735e07a3c4.png)